Terminal.Gui/Views/ComboBox.cs

```diff
@@ -81,7 +81,7 @@ private void Initialize()
 			};
 
 			// TODO: LayoutComplete event breaks cursor up/down. Revert to Application.Loaded 
-			Application.Loaded += (sender, a) => {
+			Application.Loaded += (object sender, Application.ResizedEventArgs a) => {
```

Comment:
This won't work. To prove it change your demo code in `UI Catalog` to:

```csharp
// ComboBox
var lbComboBox = new Label ("ComboBox") {
	ColorScheme = Colors.TopLevel,
	X = Pos.Right (lbListView) + 1,
	Width = Dim.Percent(30)
};

var comboBox = new ComboBox () {
	X = Pos.Right (listview) + 1,
	Y = Pos.Bottom (lbListView) + 1,
	Height = Dim.Fill (2),
	Width = Dim.Percent (30)
};
```

You'll see the combo box does not size with the parent.

![](https://i.imgur.com/gAygoiM.gif)